sui/open_rpc/samples/objects.json

```diff
@@ -101,14 +101,14 @@
         }
       },
       "owner": {
-        "AddressOwner": "0x675ced84037e1ab83177b082d89981b9ca720545"
+        "AddressOwner": "0x1234567890abcdef101112131415161718191a1b"
       },
-      "previousTransaction": "hka0OV95yi77Y5aeoOA58selKIazvhmv5EXYXM31Quk=",
+      "previousTransaction": "AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA=",
```

Comment:
this is the genesis transaction - just want to make sure the change is intentional ?